### PR TITLE
[FIX] Livechat trigger messages covering all the website

### PR DESCRIPTION
**Pull request by tiagoevanp**

### CODE REVIEW

File: packages/livechat/src/widget.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ const updateWidgetStyle = (isOpened) => {
 	}
 
 	if (isOpened) {
-		widget.style.left = smallScreen ? '0' : 'auto';
+		widget.style.left = smallScreen && widget.dataset.state !== 'triggered' ? '0' : 'auto';
```
Comment (MartinSchoeler) [inline packages/livechat/src/widget.js:84]: Could you save this value to avoid repetition? `cons isTriggered = widget.dataset.state === 'triggered'`

File: packages/livechat/package.json [hunk 2]
```diff
@@ -112,5 +112,8 @@
 		"updateFiles": [
 			"package.json"
 		]
+	},
+	"volta": {
```
Comment (MartinSchoeler) [inline packages/livechat/package.json:116]: Are these changes required for this to work or just QoL Improvements? If they are we could do this on a different PR :)

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]

File: packages/livechat/webpack.config.js [hunk 3]
```diff
@@ -177,6 +177,7 @@ module.exports = (env, argv) => [
 			clientLogLevel: 'trace',
 			overlay: true,
 			stats: 'normal',
+			writeToDisk: true,
```
Comment (MartinSchoeler) [inline packages/livechat/webpack.config.js:180]: Same question about QoL here too

**Review by MartinSchoeler — CHANGES_REQUESTED** [review]

File: packages/livechat/package.json (hunk 2 above)
Comment (tiagoevanp) [reply packages/livechat/package.json:116]: 😢 no...

File: packages/livechat/webpack.config.js (hunk 3 above)
Comment (tiagoevanp) [reply packages/livechat/webpack.config.js:180]: 😢 no...

**Review by aleksandernsilva — APPROVED** [review]

**Review by MartinSchoeler — APPROVED** [review]
